website/docs/docs/cloud/dbt-cloud-ide/ide-glossary.md

```diff
@@ -0,0 +1,245 @@
+---
+title: "IDE glossary"
+id: ide-glossary
+description: "Develop, test, run, and build in the Cloud IDE. With the Cloud IDE, you can compile dbt code into SQL and run it against your database directly"
+sidebar_label: IDE glossary
+tags: [IDE]
+---
+
+The [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) is an interface tool for developers to effortlessly build, test, run, and version-control their dbt projects - all from the convenience of your browser. Use the Cloud IDE to compile dbt code into SQL and run it against your database directly -- no command line required!
+
+## Landing page
+
+The IDE is designed to streamline your workflow, and features a popular user interface and layout with all files and folders on the left and the editor on the right. 
+
+The basic layout of the IDE landing page is divided into the following areas:
+
+A &mdash; [Git repository link](#git-repository-link)<br />
+B &mdash; [Documentation site button](#documentation-site-button)<br />
+C &mdash; [Version Control Menu](#version-control-menu)<br />
+D &mdash; [File Explorer](#file-explorer)<br />
+E &mdash; [Command bar](#command-bar)<br />
+F &mdash; [IDE Status button](#ide-status-button)<br />
+G &mdash; [Create New File button](#create-new-file-button)<br />
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-landing.jpg" width="100%" title="The Cloud IDE landing page"/>
+
+
+#### Git Repository link 
+The Git Repository Link displays the name of the active branch. Clicking this element will hyperlink to your repository, pinned to that same branch.
```

Comment:
This is not true in 100% of cases (In particular, we cannot link to Managed Repositories, I think @StefanDBTLabs would know more about the edgecases here?)